src/runCommand.ts

```diff
@@ -11,19 +12,26 @@ const spawn = require('cross-spawn');
  * Anything received on STDERR is assumed to indicate a problem, and is tracked
  * as an error.
  *
- * @param {string} command - The command to run
- * @param {Array<string>} [args] - The arguments to pass to the command
- * @returns {Array<string>} Lines of output received via STDOUT
+ * @param command - The command to run
+ * @param args - The arguments to pass to the command
+ * @returns Lines of output received via STDOUT
  */
-async function runCommand(command, args) {
-  const output = [];
+export default async function runCommand(
+  command: string,
+  args?: readonly string[],
+) {
+  const output: string[] = [];
   let mostRecentError;
   let errorSignal;
   let errorCode;
   const internalError = new Error('Internal');
   try {
-    await new Promise((resolve, reject) => {
-      const childProcess = spawn(command, args, { encoding: 'utf8' });
```

Comment:
Whoops! Not sure how that `encoding` option snuck in there. That seems to not exist.